test/unittest/bitbucket_test.go

```diff
@@ -0,0 +1,56 @@
+package unittest
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBitbucketVariablesPopulatedWithValidValues(t *testing.T) {
+	t.Parallel()
+
+	tfOptions := GenerateTFOptions(BitbucketCorrectVariables, t, "products/bitbucket")
+	plan := terraform.InitAndPlanAndShowWithStruct(t, tfOptions)
+
+	// verify Bitbucket
+	bitbucketKey := "helm_release.bitbucket"
+	terraform.RequirePlannedValuesMapKeyExists(t, plan, bitbucketKey)
+	bitbucket := plan.ResourcePlannedValuesMap[bitbucketKey]
+	assert.Equal(t, "deployed", bitbucket.AttributeValues["status"])
+	assert.Equal(t, "bitbucket", bitbucket.AttributeValues["chart"])
+	assert.Equal(t, "https://atlassian.github.io/data-center-helm-charts", bitbucket.AttributeValues["repository"])
+}
+
+// Variables
+var BitbucketCorrectVariables = map[string]interface{}{
+	"environment_name": "dummy-environment",
+	"namespace":        "dummy-namespace",
+	"eks": map[string]interface{}{
+		"kubernetes_provider_config": map[string]interface{}{
+			"host":                   "dummy-host",
+			"token":                  "dummy-token",
+			"cluster_ca_certificate": "dummy-certificate",
+		},
+		"cluster_security_group": "dummy-sg",
+	},
+	"vpc": VpcDefaultModuleVariable,
+	//"pvc_claim_name":          "dummy_pvc_claimname",
+	"db_major_engine_version": "13",
+	"db_allocated_storage":    5,
+	"db_instance_class":       "dummy_db_instance_class",
+	"db_iops":                 1000,
+	//"admin_username":          "dummy_admin_username",
+	//"admin_password":          "dummy_admin_password",
+	//"admin_display_name":      "dummy_admin_display_name",
+	//"admin_email_address":     "dummy_admin_email_address",
```

Comment:
We should delete these comments

Reply:
I cant remember from the conversation we had, but was there a reason to comment these out? BB supports credential pre-seeding so these should "just" work? If not Is there a ticket to re-instate admin credential pre-seeding?

Reply:
I though we mentioned BB does not support system admin pre-seeding. If it does, so make sense to uncomment this part and make sure it works before merge the PR.

Reply:
BB can be fully pre-seeded which includes admin creds: https://github.com/atlassian/data-center-helm-charts/blob/419a67465de0f62ec9f81b3b71223d62b189274c/src/main/charts/bitbucket/values.yaml#L540

Reply:
In this case we should not comment or remove those lines. We should add system admin credential variables for BB in root, and pass them into the BB module and apply in helm_release, the same as we have for Bamboo.

Reply:
yes, we should. But the conversation we had was to merge this PR first, and I can work on sysAdmin afterwards, so that other stories can be unblocked.

Reply:
Added in sysadminCredentials.